app/src/main/java/de/tum/in/tumcampusapp/component/tumui/calendar/CalendarActivity.java

```diff
@@ -269,7 +269,7 @@ private void refreshWeekView() {
             // Lets change some dimensions to best fit the view.
             mWeekView.setTextSize((int) TypedValue.applyDimension(TypedValue.COMPLEX_UNIT_SP, 16, getResources().getDisplayMetrics()));
             mWeekView.setEventTextSize((int) TypedValue.applyDimension(TypedValue.COMPLEX_UNIT_SP, 12, getResources().getDisplayMetrics()));
-            mWeekView.setXScrollingSpeed(0.4f);
+            mWeekView.setXScrollingSpeed(0.5f);
```

Comment:
any more info on this change?

Reply:
Oops, somehow slipped in there. I was just testing out something for myself and forgot to remove it.